src/sentry/scripts/similarity/index.lua

```diff
@@ -631,6 +652,137 @@ local commands = {
                 end
             end
         end
+    ),
+    IMPORT = takes_configuration(
+        --[[
+        Loads data returned by the ``EXPORT`` command into the location
+        specified by the ``index`` and ``key`` arguments. Data can be loaded
+        into multiple indices or keys by providing additional arguments.
+
+        If the destination specified by ``index`` and ``key`` does not exist
+        (relocating data to a new key, for example), it will be created. If
+        data already exists at the new destination, the imported data will be
+        appended to the existing data.
+        ]]--
+        function (configuration, arguments)
+            local entries = build_variadic_argument_parser({
+                {'index', identity},
+                {'key', identity},
+                {'data', function (value)
+                    return cmsgpack.unpack(value)
+                end}
```

Comment:
consistency nit: some spots have trailing comma in array-like-tables, some spots don't...i think this is the outlier in not having it, not sure if you've made a conscious decision one way or the other